communication/src/TlsConnectionManager.cpp

```diff
@@ -257,7 +258,10 @@ void ConnectionManager::startClientSSLHandshake(asio::ip::tcp::socket&& socket,
 
 void ConnectionManager::accept() {
   acceptor_.async_accept(asio::bind_executor(strand_, [this](asio::error_code ec, asio::ip::tcp::socket sock) {
-    if (ec) {
+    if (bft::communication::CommStateControl::instance().getBlockNewConnectionsFlag()) {
```

Comment:
so, if an operator starts after config update then it will not be able to initiate a new connection. Even without this change this can result in crash. 
Do we need to mention in runbook that new session from operator won't be accepted after configuration update?

Reply:
The client scaling command does not restart the operator exactly for this reason.
I think it should be fine
The only problem is if the operator is restarting after the configuration change